BarrysPeanutsAkkaCascading/src/main/java/barryspeanuts/actor/CustomerActor.java

```diff
@@ -0,0 +1,82 @@
+package barryspeanuts.actor;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.AbstractBehavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.Receive;
+import barryspeanuts.msg.PurchaseItem;
+import java.util.ArrayList;
+import java.util.Date;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CustomerActor extends AbstractBehavior<Object> {
+  Logger logger = LoggerFactory.getLogger(ShoppingCartActor.class);
+
+  private CustomerActor(ActorContext<Object> context) {
+    super(context);
+  }
+
+  public static Behavior<Object> create() {
+    return Behaviors.setup(CustomerActor::new);
+  }
+
+  public static Behavior<Object> behavior() {
+    return Behaviors.setup(CustomerActor::new);
+  }
+
+  @Override
+  public Receive<Object> createReceive() {
+    return newReceiveBuilder()
+        .onMessage(PaymentActor.PaymentReceipt.class, this::handlePaymentReceipt)
+        .onMessage(ShipperActor.ShippingReceipt.class, this::handleShippingReceipt)
+        .build();
+  }
+
+  private Behavior<Object> handlePaymentReceipt(PaymentActor.PaymentReceipt msg) {
+    String fullName =
+        String.format("%s %s", msg.getCustomer().getFirstName(), msg.getCustomer().getLastName());
+    logger.info(
+        "Customer {} processed PaymentInfo Receipt with ID: {} on {} with CC Number: {} for the amount of: {}",
+        fullName,
```

Comment:
```suggestion
    logger.info(
        "Customer {} {} processed PaymentInfo Receipt with ID: {} on {} with CC Number: {} for the amount of: {}",
        msg.getCustomer().getFirstName(),
        msg.getCustomer().getLastName(),
```

Makes no sense to preformat a string before sending to a format-supported call unless I'm missing something

Reply:
Strange, I thought I caught that one when I did a replace on all instances

Reply:
The cascading code has been removed from the project, @cretz .